### PR TITLE
Bypassing detune check when detuning by steps

### PR DESCRIPTION
**Pull request by lisazacarias**

### CODE REVIEW

**Comment by lisazacarias** [comment]
Dependency on [lcls tools update](https://github.com/slaclab/lcls-tools/pull/133) satisfied

**Comment by lisazacarias** [comment]
@nneveu @s-aderhold

File: park_linac.py [hunk 1]
```diff
@@ -179,7 +186,9 @@ def delta_func():
                 raise CavityHWModeError(f"{self} not Online, Maintenance, or Ready")
 
             self.check_resonance()
-            self.steppertuner.move_to_cold_landing(count_current=count_current)
+            self.steppertuner.move_to_cold_landing(
+                count_current=count_current, check_detune=False
+            )
```
Comment (nneveu) [inline park_linac.py:196]: I think I get that you are using false/true when you want to check for detune? 
Maybe add a comment or two in the if sections on why you check or don't for certain situations?

Comment (lisazacarias) [reply park_linac.py:196]: Do you think this explanation is clear?

**Review by nneveu — APPROVED** [review]
Thanks for the comment!

**Pull request merged**